src/core/dbus-cgroup.c

```diff
@@ -678,6 +679,28 @@ int bus_cgroup_set_property(
 
                 return 1;
 
+        } else if (streq(name, "CPUQuotaPeriodUSec")) {
+                uint64_t u64;
+
+                r = sd_bus_message_read(message, "t", &u64);
```

Comment:
so, we usually use the the maximum value of the type as indicator for "unset", hence we should refuse u64 == 0 here (as zero-length periods are not defined)

Reply:
Sounds good. But is it OK for me to keep using `parse_time` to parse the time from the config file? That one accepts 0 as a valid value... What should I do with that case?

Reply:
for the unit files, filter it out right after calling parse_time(). for the bus API filter out server side where we receive the time value, i.e. at this place here.